server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -116,7 +119,9 @@ export class ScribeLambdaFactory extends EventEmitter implements IPartitionLambd
                 await this.messageCollection.find({ documentId, tenantId }, { "operation.sequenceNumber": 1 });
             opMessages = dbMessages.map((message) => message.operation);
         } catch (error) {
-            context.log?.error(`Scribe lambda creation failed. Exception: ${inspect(error)}`);
+            const errorMessage = `Scribe lambda creation failed. Exception: ${inspect(error)}`;
+            context.log?.error(errorMessage, { messageMetaData });
+            Lumberjack.error(errorMessage, { messageMetaData });
```

Comment:
Extract error into the `exception` property and also use `getLumberBaseProperties` instead of `messageMetaData`